tests/integrationtest/r/planner/core/plan_cost_ver2.result

```diff
@@ -78,91 +78,91 @@ IndexLookUp_7	100.00	1024158.30	root
+    └─TableReader_12	100.00	262710.77	root		data:TableFullScan_11
+      └─TableFullScan_11	100.00	250038.77	cop[tikv]	table:t	keep order:false
 explain format='verbose' select /*+ use_index(t, primary), stream_agg() */ sum(a), sum(a+2) from t group by b;
 id	estRows	estCost	task	access object	operator info
-StreamAgg_8	100.00	79225.54	root		group by:Column#8, funcs:sum(Column#6)->Column#4, funcs:sum(Column#7)->Column#5
-└─Projection_17	100.00	79225.54	root		cast(planner__core__plan_cost_ver2.t.a, decimal(10,0) BINARY)->Column#6, cast(plus(planner__core__plan_cost_ver2.t.a, 2), decimal(20,0) BINARY)->Column#7, planner__core__plan_cost_ver2.t.b->Column#8
-  └─Sort_13	100.00	69195.64	root		planner__core__plan_cost_ver2.t.b
-    └─TableReader_12	100.00	35402.80	root		data:TableFullScan_11
-      └─TableFullScan_11	100.00	22730.80	cop[tikv]	table:t	keep order:false
+StreamAgg_8	100.00	306533.51	root		group by:Column#8, funcs:sum(Column#6)->Column#4, funcs:sum(Column#7)->Column#5
+└─Projection_17	100.00	306533.51	root		cast(planner__core__plan_cost_ver2.t.a, decimal(10,0) BINARY)->Column#6, cast(plus(planner__core__plan_cost_ver2.t.a, 2), decimal(20,0) BINARY)->Column#7, planner__core__plan_cost_ver2.t.b->Column#8
+  └─Sort_13	100.00	296503.61	root		planner__core__plan_cost_ver2.t.b
+    └─TableReader_12	100.00	262710.77	root		data:TableFullScan_11
+      └─TableFullScan_11	100.00	250038.77	cop[tikv]	table:t	keep order:false
 explain format='verbose' select /*+ use_index(t, primary), stream_agg() */ sum(a), sum(a+2), sum(a+4) from t group by b;
 id	estRows	estCost	task	access object	operator info
-StreamAgg_8	100.00	84215.54	root		group by:Column#10, funcs:sum(Column#7)->Column#4, funcs:sum(Column#8)->Column#5, funcs:sum(Column#9)->Column#6
-└─Projection_17	100.00	84215.54	root		cast(planner__core__plan_cost_ver2.t.a, decimal(10,0) BINARY)->Column#7, cast(plus(planner__core__plan_cost_ver2.t.a, 2), decimal(20,0) BINARY)->Column#8, cast(plus(planner__core__plan_cost_ver2.t.a, 4), decimal(20,0) BINARY)->Column#9, planner__core__plan_cost_ver2.t.b->Column#10
-  └─Sort_13	100.00	69195.64	root		planner__core__plan_cost_ver2.t.b
-    └─TableReader_12	100.00	35402.80	root		data:TableFullScan_11
-      └─TableFullScan_11	100.00	22730.80	cop[tikv]	table:t	keep order:false
+StreamAgg_8	100.00	311523.51	root		group by:Column#10, funcs:sum(Column#7)->Column#4, funcs:sum(Column#8)->Column#5, funcs:sum(Column#9)->Column#6
+└─Projection_17	100.00	311523.51	root		cast(planner__core__plan_cost_ver2.t.a, decimal(10,0) BINARY)->Column#7, cast(plus(planner__core__plan_cost_ver2.t.a, 2), decimal(20,0) BINARY)->Column#8, cast(plus(planner__core__plan_cost_ver2.t.a, 4), decimal(20,0) BINARY)->Column#9, planner__core__plan_cost_ver2.t.b->Column#10
+  └─Sort_13	100.00	296503.61	root		planner__core__plan_cost_ver2.t.b
+    └─TableReader_12	100.00	262710.77	root		data:TableFullScan_11
+      └─TableFullScan_11	100.00	250038.77	cop[tikv]	table:t	keep order:false
```

Comment:
is there any specific test case this pull request is aimed at? by now it's old tests modification, which is appreciated and help me to understand the changes in pkg/planner/core/find_best_task.go

Reply:
I added new testcase TestTableScanCostWithForce